src/config.zig

```diff
@@ -277,7 +277,7 @@ pub const configs = struct {
             .verify = true,
         },
         .cluster = .{
-            .clients_max = 4 + 3,
+            .clients_max = 4 + 4,
```

Comment:
:thinking: There's no particular reason to require the number of clients to be even... Instead, in `cli.zig`:

```
    .limit_pipeline_requests = vsr.stdx.div_ceil(constants.clients_max, 2) -|
        constants.pipeline_prepare_queue_max,
```

That will also fix the replica test failure.

Reply:
I've done this.